config/foreman.hiera/scenario/katello.yaml

```diff
@@ -0,0 +1 @@
+katello::params::enable_katello_agent: "%{alias('foreman_proxy_content::enable_katello_agent')}"
```

Comment:
Should we put this in `common.yaml`? It doesn't hurt to have it there if the katello module isn't enabled and if any other scenario would have it, we wouldn't have to keep track of it. It also saves adding another layer to Hiera.